src/test/resources/com/puppycrawl/tools/checkstyle/ant/inputs/InputCheckstyleAntTaskError.java

```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.ant;
+package com.puppycrawl.tools.checkstyle.ant.inputs;
```

Comment:
If we are following how we are organizing checks in other areas in #3891, these should be in sub-folder `checkstyleanttask`.